app/views/content/cookies.html.erb

```diff
@@ -122,6 +122,7 @@
           24 hours
         </td>
       </tr>
+      <% if @cookie_ga_code %>
```

Comment:
could we not delete this section entirely 🤔 ?

Reply:
just to confirm @alexchristensenedu we still need GA in Apply? If not I think we should remove everywhere

Reply:
@steventux @cwrw Caught up with Steve. We will only switch off GA from Manage and leave the Apply GA stuff intact